src/instructlab/model/chat.py

```diff
@@ -61,6 +62,30 @@
 
 PROMPT_PREFIX = ">>> "
 
+DEFAULT_ENDPOINT = "http://127.0.0.1:8000/v1"
```

Comment:
There's already code and constants that can be used to produce the string (`get_api_base`, `host_port` default value in `_serve` class.) These should be reused.

Reply:
Good call, I'll update this to use that instead.